tests/unittests/test_ds_identify.py

```diff
@@ -354,9 +363,18 @@ def write_mock(data):
         with open(wrap, "w") as fp:
             fp.write("\n".join(head + mocklines + endlines) + "\n")
 
+        detect_virt = None
+        for mock in [*mocks, *default_mocks]:
```

Comment:
aside:

a dict of dicts would have been the perfect data structure for a collection of unique mocks, but instead this test uses a list of dicts so here we are iterating over the list (like a bunch of other places in the code). I'm *not* going to refactor this giant test suite, so for now I just complain <rant/>